### PR TITLE
Ipyplotly test fixes

### PR DESCRIPTION
**Pull request by Kully**
Attempting to fix minor test failures in  https://github.com/plotly/plotly.py/pull/942

### CODE REVIEW

**Comment by Kully** [comment]
@chriddyp @jmmease

File: ipyplotly/.gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+*.egg-info/
```
Comment (chriddyp) [inline ipyplotly/.gitignore]: @Kully - It looks like you committed a lot of extra stuff, like this `ipyplotly` folder. Was that intentional? If not, could you rebase and open up a new PR?

Comment (chriddyp) [reply ipyplotly/.gitignore]: I highly recommend using a git client like GitHub desktop https://desktop.github.com/ to make your commits - it makes a lot easier to view what you will commit before committing it, as well as organizing your commits to only include certain files and line numbers

**Comment by chriddyp** [comment]
Also, could you make the PR against the original PR instead of `master`?

File: ipyplotly/.gitignore (hunk 1 above)
Comment (Kully) [reply ipyplotly/.gitignore]: It was intentional. `ipyplotly`can't be located without the folder in ploty.py

**Comment by Kully** [comment]
Closing in favor of https://github.com/plotly/plotly.py/pull/975

**Pull request closed without merging**